src/setup-utils/vue-config-utils.js

```diff
@@ -68,7 +69,7 @@ function baseChainWebpack(config) {
     .plugin('LanguagesPlugin')
     .use(webpack.ContextReplacementPlugin, [
       /highlight\.js\/lib\/languages$/,
-      new RegExp(`/(${[builtinLanguages, envLanguages].join('|')})$`),
+      new RegExp(`^./(${[builtinLanguages, envLanguages].filter(Boolean).join('|')}).js$`),
```

Comment:
Was the double building of languages files caused by the addition of `envLanguages` because this pattern was too inclusive in the common case where `envLanguages` is an empty string?

Just making sure I understand what went wrong and how you fixed it.

Reply:
(If so, it sounds like that was introduced back with #551 and we just didn't notice it—whoops!)

Reply:
There are a bunch of `css.js.js` files which are old deprecated file-naming convention files. Those were however being pulled into the build as well.

The extra `envLanguages` I just noticed can be empty and fixed it. I think the more restrictive regex is what is fixing it now.